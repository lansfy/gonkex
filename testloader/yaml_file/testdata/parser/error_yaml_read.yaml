- name: simple request test
path: wrong offset
