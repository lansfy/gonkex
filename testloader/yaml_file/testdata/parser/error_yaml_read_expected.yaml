- Error: "unmarshal file testdata/parser/error_yaml_read.yaml: yaml: line 1: did not find expected '-' indicator"
